### PR TITLE
Cards vue JL

### PR DESCRIPTION
**Pull request by chosta**
More cards and victory points for law suit (any player)

### CODE REVIEW

File: src/cards/base/LightningHarvest.ts [hunk 1]
```diff
@@ -26,4 +28,13 @@ export class LightningHarvest implements IProjectCard {
     public getVictoryPoints() {
       return 1;
     }
+    public metadata: CardMetadata = {
+      cardNumber: '046',
+      requirements: CardRequirements.builder((b) => b.tag(Tags.SCIENCE, 3)),
+      renderData: CardRenderer.builder((b) => {
+        b.productionBox((pb) => pb.energy(1).megacredits(1));
+      }),
+      description: 'Requires 3 Science tags. Increase your Energy production and your MC production up one step each.',
```
Comment (nwai90) [inline src/cards/base/LightningHarvest.ts:37]: Minor: `"up"` is probably redundant here

File: src/cards/venusNext/JetStreamMicroscrappers.ts [hunk 2]
```diff
@@ -70,4 +72,18 @@ export class JetStreamMicroscrappers implements IActionCard, IProjectCard, IReso
...
+      cardNumber: '234',
+      renderData: CardRenderer.builder((b) => {
+        b.effectBox((eb) => {
+          eb.titanium(1).startAction.floaters(2);
+          eb.description('Action: Spend 1 titanium to add 2 Floaters here');
+        }).br;
+        b.or().br;
+        b.effectBox((eb) => {
+          eb.floaters(2).startAction.venus(1);
+          eb.description('Action: spend 2 Floaters here to raise Venus 1 step');
```
Comment (nwai90) [inline src/cards/venusNext/JetStreamMicroscrappers.ts:85]: Minor: Capitalization for `spend`

File: src/cards/ares/LavaFlowsAres.ts [hunk 3]
```diff
@@ -2,8 +2,19 @@ import {CardName} from '../../CardName';
...
 export class LavaFlowsAres extends LavaFlows {
   public name = CardName.LAVA_FLOWS_ARES;
   public adjacencyBonus: IAdjacencyBonus = {bonus: [SpaceBonus.HEAT, SpaceBonus.HEAT]}
+  public metadata: CardMetadata = {
+    cardNumber: 'A11',
+    renderData: CardRenderer.builder((b) => {
+      b.temperature(2).br;
+      b.tile(TileType.LAVA_FLOWS, false, true);
+    }),
+    description: 'Raise temperature 2 steps and place this tile ON EITHER THARSIS THOLUS, ASCRAEUS MONS, PAVONIS MONS OR ARSIA MONS. This tile grants an adjacency bonus of 2 heat.',
```
Comment (kberg) [inline src/cards/ares/LavaFlowsAres.ts:18]: capitalize "adjacency bonus"?

File: src/cards/base/Livestock.ts [hunk 4]
```diff
@@ -34,5 +38,24 @@ export class Livestock implements IActionCard, IProjectCard, IResourceCard {
...
+        b.productionBox((pb) => {
+          pb.minus().plants(1).nbsp.plus().megacredits(2);
+        }).br;
+        b.text('1 VP for each Animal on this card.', CardRenderItemSize.TINY, true);
+      }),
+      description: {
+        text: 'Requires 9% oxygen. Decrease your Plant production 1 step and increase your MC production 2 steps',
+        align: 'left',
+      },
+      victoryPoints: CardRenderDynamicVictoryPoints.animals(1, 1),
```
Comment (kberg) [inline src/cards/base/Livestock.ts:58]: minor: you keep description at the bottom, but victory points below even that. VP up above description?

File: src/cards/prelude/LavaTubeSettlement.ts [hunk 5]
```diff
@@ -40,4 +42,16 @@ export class LavaTubeSettlement implements IProjectCard {
...
+    public metadata: CardMetadata = {
+      cardNumber: 'P37',
+      renderData: CardRenderer.builder((b) => {
+        b.productionBox((pb) => {
+          pb.minus().energy(1).br;
+          pb.plus().megacredits(2);
+        }).br;
+        b.city().asterix();
+      }),
+      description: 'Decrease your energy production 1 step and increase your MC production 2 steps. Place a City Tile on a VOLCANIC AREA regardless of adjacent cities.',
```
Comment (kberg) [inline src/cards/prelude/LavaTubeSettlement.ts:55]: capitalize energy.

File: src/cards/promo/LawSuit.ts [hunk 6]
```diff
@@ -32,5 +36,13 @@ export class LawSuit implements IProjectCard {
     public getVictoryPoints() {
       return -1;
     }
+    public metadata: CardMetadata = {
+      cardNumber: 'X06',
+      renderData: CardRenderer.builder((b) => {
+        b.text('steal', CardRenderItemSize.SMALL, true).megacredits(3).any.asterix();
+      }),
+      description: 'Steal 3 MC from a player that REMOVED YOUR RESOURCES OR DECREASED YOUR PRODUCTION this generation. Place this card face down in THAT PLAYER\'S EVENT PILE.',
```
Comment (kberg) [inline src/cards/promo/LawSuit.ts:44]: Damn, this is rough!

File: src/components/card/CardVictoryPoints.ts [hunk 7]
```diff
@@ -26,7 +32,7 @@ export const CardVictoryPoints = Vue.component('CardPoints', {
     },
   },
   template: `
-      <div v-if="isObject()" :class="getClasses()"><div>{{ this.victoryPoints.getPointsHtml() }}</div><CardRenderItemComponent :item="this.victoryPoints.item" /></div>
+      <div v-if="isObject()" :class="getClasses()"><div>{{ this.victoryPoints.getPointsHtml() }}</div><CardRenderItemComponent v-if="this.victoryPoints.item !== undefined" :item="this.victoryPoints.item" /></div>
```
Comment (kberg) [inline src/components/card/CardVictoryPoints.ts:35]: This should be multiple lines at this point.

**Review by kberg — APPROVED** [review]

**Comment by kberg** [comment]
Please merge yourself at your convenience, @chosta

File: src/cards/base/Livestock.ts (hunk 4 above)
Comment (chosta) [reply src/cards/base/Livestock.ts:58]: In the western world reading is from left to rigth, so  I've intuitively  put description before points :)

**Comment by chosta** [comment]
Thanks @kberg and @nwai90 for taking the time to review these not so exciting PRs

**Pull request merged**